src/common/enums/button-type/button-type.ts

```diff
@@ -0,0 +1,7 @@
+enum ButtonType {
+  TEXT = 'text',
+  CONTAINED = 'contained',
+  OUTLINED = 'outlined',
+}
+
+export { ButtonType };
```

Comment:
Let's group button-type, heading-level, text-appearance, app-color and put ts files to one folder enums/ui